koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -410,27 +415,47 @@ def convert_to_parquet(self):  # noqa: C901
 
         failed_conversion = []
         daily_data_frames = []
-        for csv_filename in self.file_list:
-            if self.provider_type == Provider.PROVIDER_OCP and self.report_type is None:
-                LOG.warn(
-                    log_json(
-                        self.tracing_id,
-                        msg="could not establish report type",
-                        context=self.error_context,
-                        filename=csv_filename,
+        # Azure and AWS should now always have split daily files
+        if self.provider_type in [
+            Provider.PROVIDER_AWS,
+            Provider.PROVIDER_AWS_LOCAL,
+            Provider.PROVIDER_AZURE,
+            Provider.PROVIDER_AZURE_LOCAL,
+        ]:
+            file_list = self.split_file_list
+        else:
+            file_list = self.file_list
+
+        if file_list:
```

Comment:
I 100% agree on readability here but since we dont return just log a warning the for is just going to cry in pain from file_list being referenced before assignment. Could change that I guess to though.